### PR TITLE
Implement Popup window

### PR DESCRIPTION
**Pull request by CindyDorantes**
First review request. Interactions implemented:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears
- Popups for both mobile and desktop screen sizes
- Use a JavaScript array to store all of the information for all projects.

Second review request. Changes introduced:
- Fix scroll
- Fix desktop design

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters errors, kindly fix this so the linter checks are green and passing

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -721,3 +721,143 @@ button:active {
...
+  justify-content: flex-start;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 4%;
+  margin: 2%;
+  transition: 200ms ease-in-out;
+  overflow-y: scroll;
+  overflow: hidden;
+}
```
Comment (keroloussamy) [inline style.css:760]: - [ ] Kindly I can't see the close button, kindly add it to match the Figma design.
- [ ] Kindly I can't see the tags and the paragraph, kindly add them to match the Figma design.
- [ ] Kindly make the pic take the whole width, to match the Figma design.

Your design
image

Figma design
image

File: style.css [hunk 2]
```diff
@@ -721,3 +721,143 @@ button:active {
...
+
```
Comment (keroloussamy) [inline style.css:761]: - [ ] Kindly for the mobile version, I can't see the whole paragraph and the two buttons. Kindly make them visible or add scroll.

Your design
image

Original design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @CindyDorantes, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @CindyDorantes 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏
 

- [ ] [optional] for tablet screen your overlay doesn't cover the whole screen so it would be better if you fix it
image


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**